pkgs/by-name/li/liboprf/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkgconf,
+  libsodium,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "liboprf";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "stef";
+    repo = "liboprf";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-auC6iVTMbLktKCPY8VgOdx2dMI2KDzNgtY1zyNXjM1A=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/src";
+
+  strictDeps = true;
+
+  nativeBuildInputs = [ pkgconf ];
+
+  buildInputs = [ libsodium ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
```

Comment:
```suggestion
  makeFlags = [ "PREFIX=${placeholder "$out"}" ];
```

Reply:
Doesn't work. `"$out"` is perfectly common and functional anyways